packages/riipen-ui/src/components/MenuList.test.jsx

```diff
@@ -0,0 +1,299 @@
+
+      const wrapper = mount(
+        <MenuList classes={classes}>
+          <MenuItem />
+        </MenuList>
+      );
+
+      expect(
+        wrapper
+          .find("MenuList")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-menulist"];
+
+      const wrapper = mount(
+        <MenuList>
+          <MenuItem />
+        </MenuList>
+      );
+
+      expect(
+        wrapper
+          .find("MenuList")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
+  });
+
+  describe("color prop", () => {
+    it("passes color prop into children nodes", () => {
+      const color = "secondary";
+
+      const wrapper = mount(
+        <MenuList color={color}>
+          <MenuItem>one</MenuItem>
+        </MenuList>
+      );
+
+      expect(wrapper.find("MenuItem").props().color).toEqual(color);
+    });
+
+    it("gives an error with invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+
+      mount(
+        <MenuList color="pink">
+          <MenuItem />
+        </MenuList>
+      );
+      expect(errors).toHaveBeenCalled();
+    });
+  });
+
+  describe("selectChange prop", () => {
+    it("invokes selectChange with a null idx when div is clicked", () => {
+      const selectChange = jest.fn();
+
+      const wrapper = mount(
+        <MenuList selectChange={selectChange}>
+          <MenuItem>one</MenuItem>
+        </MenuList>
+      );
+
+      wrapper
+        .find("MenuList")
+        .childAt(0)
+        .simulate("click");
+
+      expect(selectChange).toHaveBeenLastCalledWith(null, expect.any(Object));
+    });
+
+    describe("when variant is selection", () => {
+      it("invokes selectChange when child clicked", () => {
+        const selectChange = jest.fn();
+
+        const wrapper = mount(
+          <MenuList selectChange={selectChange} variant="selection">
+            <MenuItem disabled={false} />
+            <MenuItem />
+          </MenuList>
+        );
+
+        wrapper
+          .find("MenuItem")
+          .at(0)
+          .childAt(0)
+          .simulate("click");
+
+        expect(selectChange).toHaveBeenCalledWith(0, expect.any(Object));
+      });
+
+      it("does not invoke selectChange when child clicked and child is disabled", () => {
+        const selectChange = jest.fn();
+        const disabled = true;
+
+        const wrapper = mount(
+          <MenuList selectChange={selectChange} variant="selection">
+            <MenuItem disabled={disabled}>one</MenuItem>
+          </MenuList>
+        );
+
+        wrapper
+          .find("MenuItem")
+          .at(0)
+          .childAt(0)
+          .simulate("click");
+
+        expect(selectChange).not.toHaveBeenCalled();
+      });
+    });
+  });
+
+  describe("selectedIndex prop", () => {
```

Comment:
Should we add a test for selectedIndex + autoFocus? I don't see this combination.